### ISSUE TITLE
Suggestion : Intégration au magasin d’extensions de Firefox

### ISSUE THREAD

**Issue by Anglodel** [lovasoa/ophirofox#201]
Bonjour !
Est-il prévu tôt ou tard que l’extension soit ajoutée au magasin des addons de Firefox ? Je vois que c’est déjà le cas pour Chrome — mais c’est peut-être obligatoire avec les CGU de ce dernier.
Aussi, quelle est la raison, si elle existe, du fait que l’extension ne soit du coup pas — encore ? — disponible téléchargeable directement là-bas ? Ce serait peut-être moins inquiétant pour des utilisateurs davantage néophytes.
Bonne journée,

**Comment by lovasoa** [comment]
Bonjour et bienvenue,
  Le sujet a déjà été discuté ici: https://github.com/lovasoa/ophirofox/issues/139#issuecomment-1809264813

  Pour faire court, l'extension a été proposée, et refusée par mozilla, faute de pouvoir la tester avec un compte d'université de test.

**Comment by lovasoa** [comment]
Mise à jour (mentionné dans #139):

J'ai recontacté mozilla, voici leur réponse: 

> A limited audience add-on needs to be self hosted. It cannot be listed publicly on AMO. You can see more details here https://extensionworkshop.com/documentation/publish/submitting-an-add-on/#self-distribution
> As for the other add-on. Thank you for letting us know. We will review it accordingly.


*the other addon* fait référence à https://addons.mozilla.org/en-US/firefox/addon/ophirofox-unoficial/

**Comment by aherrou** [comment]
Peut-être faudrait-il indiquer dans le Readme la raison pour laquelle l'extension n'est pas sur AMO? Je me suis demandée un moment si c'était parce que l'extension posait des problèmes de sécurité aux yeux de Mozilla, et j'ai été rassurée en lisant cette issue, mais ça serait bien que l'info soit présente sur la page d'accueil.

**Comment by lovasoa** [comment]
Oui, ce serait bien. Et peut-être même un petit appel à fournir un compte de test ? PR welcome

**Comment by Erwyn** [comment]
Pour information, je viens d'écrire à la BnF pour faire une demande. On ne sait jamais! 🤞

**Comment by Anglodel** [comment]
> Pour information, je viens d'écrire à la BnF pour faire une demande. On ne sait jamais! 🤞

Demander quoi, exactement ! (Mais ç’a l’air super)

**Comment by Erwyn** [comment]
Un compte de test pour l'application.

**Comment by bonnebulle** [comment]
dead link
https://addons.mozilla.org/en-US/firefox/addon/ophirofox-unoficial/
il existe une extention finalement ?
officielle ou non...
En telechargeant le release mobile je ne sais pas trop quoi faire !

**Comment by Erwyn** [comment]
@lovasoa Quid de demander un compte directement à Europresse. Est-ce que l'extension va à l'encontre de leurs conditions d'utilisation ?

**Comment by lovasoa** [comment]
Ils nous détestent ^^

Ils ont envoyé un mail menaçant à toutes les universités en leur demandant de désinstaller l'extension.

**Comment by Erwyn** [comment]
Intéressant... J'imagine donc que nous n'allons pas leur demander de comptes ... :joy: . Un argumentaire a été donné ? L'extension ne sort aucun contenu, au contraire elle nous envoie sur leur site directement.

**Comment by aherrou** [comment]
> Ils nous détestent ^^
> 
> Ils ont envoyé un mail menaçant à toutes les universités en leur demandant de désinstaller l'extension.

Qu'est-ce qu'ils reprochent exactement à l'extension?

**Comment by lovasoa** [comment]
Ils ne le disent pas, mais je suppose qu'ils n'ont pas tellement intérêt à ce que leur service soit effectivement utilisé. Les universités payent autant, que les étudiants utilisent le service ou non. Mais ça doit être plus dur de négocier avec les journaux de l'autre côté quand ces derniers ont peur que leurs abonnements payants soient cannibalisés par l'utilisation d'europresse.

**Comment by Write** [comment]
@lovasoa Pourrais-tu inscrire ici une adresse e-mail à laquelle on pourrait te donner des identifiants, si une âme bienveillante souhaite bien te les partager pour une éventuelle nouvelle demain à Mozilla ?

**Comment by lovasoa** [comment]
Oui! contact@ophir.dev

**Comment by Erwyn** [comment]
hoho, on peut espérer une arrivée dans le magazin Firefox ?

**Comment by Write** [comment]
> hoho, on peut espérer une arrivée dans le magazin Firefox ?

Hello, il faudrait envoyer des identifiants EuroPresse au développeur d'Ophirofox à contact@ophir.dev pour qu'il puisse soumettre à Mozilla. Ils veulent absolument pouvoir tester l'extension avec des identifiants pour autoriser la publication de celle-ci.

**Comment by D4llo** [comment]
Cela marcherait avec des identifiants BNF?

**Comment by Erwyn** [comment]
> Hello, il faudrait envoyer des identifiants EuroPresse au développeur d'Ophirofox à contact@ophir.dev pour qu'il puisse soumettre à Mozilla.

Ah pardon je pensais que vous en aviez fourni.

> Cela marcherait avec des identifiants BNF?
 J'imagine que oui. Ces identifiants permettent amplement de tester l'application.

**Comment by D4llo** [comment]
Ca me va d'en acheter et de filer les miens du coup

**Comment by Write** [comment]
> Ca me va d'en acheter et de filer les miens du coup

Votre participation pourrait être très utile pour le projet et sa publication par Mozilla, bien que nous ne puissions garantir aucun résultat.

**Comment by D4llo** [comment]
ok je m'en occupe sous peu. on va tenter :)

**Comment by Write** [comment]
Je me permet de ré-up le sujet si quelqu'un est tjrs volontaire :)

**Comment by Write** [comment]
Ophirofox est enfin disponible sur le Firefox AMO : https://addons.mozilla.org/fr/firefox/addon/ophirofox/

Cependant je n'ai pas pu cocher la case "Compatible Mobile" pour le moment, vue qu'il faut que la compatiblité soit un seule et même .xpi (ce qui n'est pas le cas actuellement, sur github il ya un .xpi pour mobile et un pour Firefox PC). Je vais voir si on peut self sign tout en continuant d'utiliser AMO (mais il y a les soucis de délai de signing pour les extensions "selfhosted"  en ce moment ..)
